### PR TITLE
[MINVOKER-258] normalize the line endings code to LF

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

**Comment by michael-o** [comment]
Why not use `.gitattributes` for that?

**Comment by slawekjaranowski** [comment]
`.gitattribute` we use when some file should has the same line end code on all operating system.

For standard files we should have one and the same line end code in all files in repo and allow users to setup local environment by `core.autocrlf`

Now in repo we have mixed line end code.

File: src/main/resources/invoker-report_fr.properties [hunk 1]
```diff
@@ -15,21 +15,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-report.invoker.result.description   = R�sultat des invocations de Maven
-report.invoker.result.name          = Invoker R�sultat de builds
+report.invoker.result.description   = R�sultat des invocations de Maven
```
Comment (pzygielo) [inline src/main/resources/invoker-report_fr.properties:18]: I don't know any French, but in my diff view this `é` -> `�` doesn't look right.

Comment (slawekjaranowski) [reply src/main/resources/invoker-report_fr.properties]: @pzygielo you right - the file is encoded by iso-8859-2, probably special language chars should be escaped by unicode \uxxxx  - but it is another issue
I removed this file from changeset

**Comment by slawekjaranowski** [comment]
What we do with this PR?

**Comment by michael-o** [comment]
I need to look at this. I would prefer autodetection and line endings according to the OS by Git unless the fileformat requires LF.

**Comment by slawekjaranowski** [comment]
Exactly for this is PR ... 
in repository we should have every file with the same line encoding and conversion should be done on client site according to `core.autocrlf` settings.

**Comment by michael-o** [comment]
Alright, I will try to review this weekend.

**Comment by jira-importer** [comment]
Resolve #615

**Pull request closed without merging**